nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs

```diff
@@ -201,5 +210,31 @@ private void UpdateView()
                 }
             }
         }
+
+        protected virtual void SetOptionTag(ItemBase itemBase)
```

Comment:
There are two functions that do the same thing.

Reply:
Are they both in `EquipmentSlot` and `ItemView`? They are separated classes, but they do similar things.
Then how about making `EquipmentSlot` derives `ItemView`?

Reply:
How about changing it like this?
```
 public static bool TryGetOptionTag(ItemBase itemBase, out string tag)
        {
            tag = string.Empty;

            if (!(itemBase is Equipment equipment))
            {
                return false;
            }

            var optionCount = equipment.GetOptionCount();
            if (optionCount <= 0)
            {
                return false;
            }

            var sb = new StringBuilder();
            for (var i = 0; i < optionCount; ++i)
            {
                sb.AppendLine("<sprite name=UI_icon_option>");
            }
            tag = sb.ToString();
            return true;
        }
```

Reply:
Seems like a good idea.